src/v2/Apps/Consign/Routes/MarketingLanding/Components/__tests__/ContactUs.jest.tsx

```diff
@@ -1,34 +1,26 @@
-import { mount } from "enzyme"
 import { ContactUs } from "../ContactUs"
 import { MockBoot } from "v2/DevTools"
 import { Breakpoint } from "@artsy/palette"
+import { render, screen } from "@testing-library/react"
 
 jest.mock("react-tracking")
 
 describe("ContactUs", () => {
   const getWrapper = (breakpoint = "md") => {
-    return mount(
+    return render(
       <MockBoot breakpoint={breakpoint as Breakpoint}>
         <ContactUs />
       </MockBoot>
     )
   }
 
-  it("contains correct email in body", () => {
-    const wrapper = getWrapper()
-    expect(wrapper.text()).toContain(
-      "Email us at consign@artsy.net or call +1-646-797-3423 for more information on how Artsy can sell your artwork."
-    ) // pull text minus divs
-  })
-
-  it("has correct email link", () => {
-    const wrapper = getWrapper()
-    expect(wrapper.find("a").first().props().href).toBe(
-      "mailto:consign@artsy.net"
-    )
-    const wrapperMobile = getWrapper("xs")
-    expect(wrapperMobile.find("a").first().props().href).toBe(
-      "mailto:consign@artsy.net"
-    )
+  it("contains correct email in body with, correct email link and phone number", () => {
+    getWrapper()
+    // console.log("*****************************")
+    // const text = screen.getByTestId("contactUsBody")
+    // debug(text)
+    // expect(within(text).getByText("Email us at")).toBeInTheDocument()
```

Comment:
👋

Reply:
This was just @The-Beez-Kneez passing some wrap up stuff to me. Don't worry @damassi !! 😆

Reply:
That is there because @laurabeth wanted to tinker with it and should be pushing up some work. I should have labelled it.